ingest/bulk_upload.py

```diff
@@ -20,6 +20,7 @@ def upload_entries(es, entries):
     for entry in entries:
         entry["_index"] = INDEX_NAME
         entry["_type"] = TYPE_NAME
+        entry["completions"] = [entry["cf"],entry["gw"]]
```

Comment:
Very minor style suggestion:
```suggestion
        entry["completions"] = [entry["cf"], entry["gw"]]
```